src/fides/api/graph/config.py

```diff
@@ -647,6 +653,31 @@ def serialize_erase_after(self, erase_after: Set[CollectionAddress]) -> Set[str]
         },
     )
 
+    @field_validator("partitioning")
+    @classmethod
+    def validate_partitioning(
```

Comment:
I think this location makes sense given how we're leaving it flexible and open-ended right now -